examples/mother/lib.rs

```diff
@@ -0,0 +1,219 @@
+//! # Mother of All Contracts
+//!
+//! This contracts is intented to
+//! demonstrate rich I/O posibilities of ink! contracts, namely:
```

Comment:
Is no longer only about I/O but every functionality that is important for off chain interaction.